src/lib/reticket.js

```diff
@@ -33,6 +33,7 @@ export async function reticketPointBetweenWallets({
   contracts,
   onUpdate,
   transferEth = false,
+  nextNetworkKeyRevision = INITIAL_NETWORK_KEY_REVISION,
```

Comment:
another function somewhere takes `revision` as the argument name — can we standardize on `networkKeyRevision` or `revision` for arguments?